src/shammodels/ramses/include/shammodels/ramses/SolverConfig.hpp

```diff
@@ -86,7 +86,6 @@ namespace shammodels::basegodunov {
 
         inline bool is_gas_passive_scalar_on() { return npscal_gas == 0; }
     };
```

Comment:
```suggestion
    };

```
To be sure that no extra changes are committed you can check on the "files changed" tab the list of changes to clean up the PR.